packages/babel-preset/README.md

```diff
@@ -101,7 +101,7 @@ module.exports = {
       {
         evaluationRules: [
           {
-            action: require('@linaria/shaker').default,
+            action: require('@griffel/babel-preset').shakerEvaluator,
```

Comment:
should we point to linaria documentation how to configure this ?

Reply:
Well, there is no way to configure `@linaria/shaker` so I am not sure what to link there...